pubsub/api/PubsubSample/Program.cs

```diff
@@ -278,6 +278,9 @@ public static object CreateTopic(string projectId, string topicId)
             // batches.
             foreach (string text in messageTexts)
             {
+                // Record all publishing Tasks. When each Task completes
+                // without error, the message has been successfully published.
+                // In real use, don't simply store all publish Tasks forever.

```

Comment:
Please update the code comment with an answer to the question:
What should the user do "in real use"?

Reply:
Done. PTAL